test/unit/rules/eol-last-test.js

```diff
@@ -43,6 +43,19 @@ generateRuleTests({
       config: 'always',
       template: '{{#my-component}}{{/my-component}}\n',
     },
+    {
```

Comment:
Can you add a comment in the test case explaining why the rule is off in this case.